src/opm/parser/eclipse/EclipseState/InitConfig/FoamConfig.cpp

```diff
@@ -128,12 +161,23 @@ FoamConfig::FoamConfig(const Deck& deck)
     }
 }
 
+FoamConfig::FoamConfig(const std::vector<FoamData>& data)
+    : data_(data)
+{
+}
+
 const FoamData&
 FoamConfig::getRecord(std::size_t index) const
 {
     return this->data_.at(index);
 }
 
+const std::vector<FoamData>&
+FoamConfig::records() const
+{
+    return data_;
```

Comment:
Don't like `this->`do you?

Reply:
i'm used to the rule 'this for methods, no this for data' in other projects. i have no opinion either way, i'll give you your this no problem.